gtest/src/system.rs

```diff
@@ -97,4 +98,12 @@ impl System {
         let code = fs::read(&path).unwrap_or_else(|_| panic!("Failed to read file {:?}", path));
         self.0.borrow_mut().store_new_code(&code)
     }
+
+    pub fn get_mailbox<ID: Into<ProgramIdWrapper>>(&'_ self, id: ID) -> Mailbox<'_> {
+        let program_id = id.into().0;
+        if self.0.borrow_mut().actors.contains_key(&program_id) {
+            panic!("Such program id is already in actors list");
+        }
+        Mailbox::new(program_id, &self.0)
```

Comment:
empty mailbox with such program_id should be created in manager.mailbox to correct work of `free_id_nonce` and infallibility of getting mailbox for existing Mailbox structure